README.md

```diff
@@ -104,7 +104,7 @@ The following constants are reported as the only parameter to the
 
 ### Methods
 
-- `media.getCurrentAmplitude`: Returns the current position within an audio file.
+- `media.getCurrentAmplitude`: Returns the current amplitude of the current recording.
```

Comment:
Because that definition is also wrong, you are not recording, so the "of the current recording" makes no sense. And all the other function definitions say "within an audio file" or "of an audio file"